plugins/chat/assets/javascripts/discourse/components/user-threads/index.gjs

```diff
@@ -41,14 +44,23 @@ export default class UserThreads extends Component {
     >
       <list.Item as |thread|>
         <div class="c-user-thread" data-id={{thread.id}}>
-          <ThreadTitle @thread={{thread}} />
+          {{#if this.site.mobileView}}
+            <ChannelIcon @thread={{thread}} />
+          {{/if}}
+
           <ChannelTitle @channel={{thread.channel}} />
-          <ThreadIndicator
-            @message={{thread.originalMessage}}
-            @interactiveUser={{false}}
-            @interactiveThread={{false}}
-            tabindex="-1"
-          />
+          <ThreadTitle @thread={{thread}} />
+
+          {{#if this.site.mobileView}}
+            <ThreadPreview @preview={{thread.preview}} />
```

Comment:
Handled via a separate component now as want to treat the thread layout differently on the My Threads page on mobile. Everywhere else on mobile/desktop we still use the boxed thread layout within `ThreadIndicator`.